src/elements/content-sidebar/MetadataSidebarRedesign.tsx

```diff
@@ -118,23 +125,41 @@ function MetadataSidebarRedesign({
             sidebarView={SIDEBAR_VIEW_METADATA}
             title={formatMessage(messages.sidebarMetadataTitle)}
         >
-            <div className="bcs-MetadataSidebarRedesign-content">
-                {errorMessageDisplay}
-                {status === STATUS.LOADING && (
-                    <LoadingIndicator aria-label={formatMessage(messages.loading)} data-testid="loading" />
-                )}
-                {showEmptyState ? (
-                    <MetadataEmptyState level={'file'} isBoxAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled} />
-                ) : (
-                    editingTemplate && (
-                        <MetadataInstanceEditor
-                            isBoxAiSuggestionsEnabled={isBoxAiSuggestionsEnabled}
-                            isUnsavedChangesModalOpen={isUnsavedChangesModalOpen}
-                            template={editingTemplate}
+            <AutofillContextProvider isAiSuggestionsFeatureEnabled={isBoxAiSuggestionsEnabled}>
+                <div className="bcs-MetadataSidebarRedesign-content">
+                    {errorMessageDisplay}
+                    {status === STATUS.LOADING && (
+                        <LoadingIndicator aria-label={formatMessage(messages.loading)} data-testid="loading" />
```

Comment:
```suggestion
                        <LoadingIndicator aria-label={formatMessage(messages.loading)} />
```
shouldnt need the data-testid, should be able to target this loading indicator via getByRole for the aria-label